README.md

```diff
@@ -25,24 +26,176 @@ Spline currently supports Spark 2.2+, but in older versions (especially 2.2) lin
 
 Spline aims to fill a big gap within the Apache Hadoop ecosystem. Spark jobs shouldn’t be treated only as magic black boxes; people should be able to understand what happens with their data. Our main focus is to solve the following particular problems:
 
-- Regulatory requirement for SA banks (BCBS 239)
+-  Regulatory requirement for SA banks (BCBS 239)
 
     By 2020, all South African banks will have to be able to prove how numbers are calculated in their reports to the regulatory authority.
 
-- Documentation of business logic
+-  Documentation of business logic
 
     Business analysts should get a chance to verify whether Spark jobs were written according to the rules they provided. Moreover, it would be beneficial for them to have up-to-date documentation where they can refresh their knowledge of a project.
 
-- Identification of performance bottlenecks
+-  Identification of performance bottlenecks
 
     Our focus is not only business-oriented; we also see Spline as a development tool that should be able to help developers with the performance optimization of their Spark jobs.
 
+---
+
+## Get Spline
+To get started, you need to get a minimal set of Spline's moving parts - 
+a server, an admin tool and a client Web UI to see the captured lineage.
+
+There are two ways how to do it:
+
+#### Download prebuild Spline artifacts from the Maven repo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4568241)